vendor/cloud.google.com/go/bigtable/cmd/cbt/cbtdoc.go

```diff
@@ -145,7 +145,11 @@ Create a table
 Usage:
 	cbt createtable <table> [families=family[:gcpolicy],...] [splits=split,...]
 	  families: Column families and their associated GC policies. For gcpolicy,
+<<<<<<< HEAD
```

Comment:
This seems wrong, this is a conflict diff.

Reply:
Fixed.